src/server/models/Ride.js

```diff
@@ -1,30 +1,87 @@
-import {Ride} from '../../db/Ride';
-export const addRide = function(req, res) {
+import { Ride } from '../../db/Ride';
+
+var lyfthelper = require('./../../../../carvis/carvis-web/src/server/utils/lyft-helper.js');
+var uberhelper = require('./../../../../carvis/carvis-web/src/server/utils/uber-helper.js');
+
+export const addRide = function (req, res) {
+  var rideId;
+
   Ride.create(req.body)
-  .then((ride) => res.json(ride))
-  .catch((err) => res.json(err));
+    .then((ride) => {
+      rideId = ride.id; // is there a cleaner way to do this?
+      return res.json(ride);
+    })
+    .catch((err) => res.json(err)); // add catch for errors.
+
+  let vendor = req.body.winner;
+  let origin = {
+    lat: req.body.originLat,
+    lng: req.body.originLng,
+    routableAddress: originRoutableAddress
+  };
+  let destination = {
+    lat: req.body.destinationLat,
+    lng: req.body.destinationLng,
+    routableAddress: req.body.destinationRoutableAddress
+  };
+  let partySize = req.body.partySize || 1;
+
+  // carvisUserId -- to query the user table for tokens etc.
+  let userId = req.body.userId;
+  let dbURL = 'http://54.183.205.82/users' + userId;
+
+  return fetch(dbURL, {
```

Comment:
might we wanna move this call to after successfully inserting the ride into the db? like, nested inside the .then for the Ride.create call?